docs/content/migration/v1-to-v2.md

```diff
@@ -226,6 +226,10 @@ Then, a [router's TLS field](../routing/routers/index.md#tls) can refer to one o
         # will terminate the TLS request
         [http.routers.Router-1.tls]
           options = "myTLSOptions"
+          certResolver = "myCertResolver"
```

Comment:
This example is for illustrate how to use external certificate, so it's not the good place for your example.

Reply:
Thanks for your links. When I saw `options = "myTLSOptions"`, I thought it would be appropriate to add also `certResolver = "myCertResolver"`.